gravityflow/gravityflowvacation
# Add Custom has_calculation() Method to Fix Calculations

In GravityForms 2.5, our vacation field no longer updates when calculations are applied to it. After some digging, it turns out that this is because the default `has_calculation()` method in the base `class-gf-field.php` file was only returning true for `number` fields and `calculation` fields; anything else would return false, and thus the scripts to enable calculation processing never actually fired. 

This fix simply overrides the parent `has_calculation()` method in our vacation field and returns an accurate value based on its `enableCalculation` and `calculationFormula` properties.

Addresses issue 110: https://github.com/gravityflow/backlog/issues/110

## Testing Instructions
1. Create a form, and add 2 number fields and a Vacation field
2. Enable calculations for the Vacation field, and add a formula to add the 2 number fields together.
3. Save and preview the form - enter values into the number fields and ensure that the Vacation field updates as expected.
